### PR TITLE
Add GDevelop 5 cask

### PR DESCRIPTION
**Pull request by execat**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by execat** [comment]
```
» brew style --fix Casks/gdevelop.rb
1 file inspected, no offenses detected

» brew audit --cask Casks/gdevelop.rb
audit for gdevelop: passed

» brew install --cask Casks/gdevelop.rb
==> Downloading https://github.com/4ian/GDevelop/releases/download/v5.0.131/GDevelop-5-5.0.131.dmg
Already downloaded: /Users/atm/Library/Caches/Homebrew/downloads/a02d5cff49b3a4f43e1ef4b20741b7e38df17777ecce839847e5c5c8735ac4c9--GDevelop-5-5.0.131.dmg
==> Installing Cask gdevelop
==> Moving App 'GDevelop 5.app' to '/Applications/GDevelop 5.app'
🍺  gdevelop was successfully installed!

» brew uninstall --cask Casks/gdevelop.rb
==> Uninstalling Cask gdevelop
==> Backing App 'GDevelop 5.app' up to '/usr/local/Caskroom/gdevelop/5.0.131/GDevelop 5.app'
==> Removing App '/Applications/GDevelop 5.app'
==> Purging files for version 5.0.131 of Cask gdevelop
```

File: Casks/gdevelop.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+cask "gdevelop" do
+  def major_version
+    "5"
+  end
+
+  version "5.0.131"
+  sha256 "be81dc06ba56333cffb95ff86ea64066b1acbddb09ddd2f74a841759e9a00b53"
+
+  url "https://github.com/4ian/GDevelop/releases/download/v#{version}/GDevelop-#{major_version}-#{version}.dmg",
```
Comment (SMillerDev) [inline Casks/gdevelop.rb:9]:
```suggestion
  url "https://github.com/4ian/GDevelop/releases/download/v#{version}/GDevelop-#{version.major}-#{version}.dmg",
```

File: Casks/gdevelop.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+      verified: "github.com/4ian/GDevelop/"
+  name "GDevelop"
+  desc "Open-source, cross-platform game engine designed to be used by everyone"
+  homepage "https://gdevelop.io/"
+
+  app "GDevelop #{major_version}.app"
```
Comment (SMillerDev) [inline Casks/gdevelop.rb:15]:
```suggestion
  app "GDevelop #{version.major}.app"
```

File: Casks/gdevelop.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+cask "gdevelop" do
+  def major_version
+    "5"
+  end
```
Comment (SMillerDev) [inline Casks/gdevelop.rb:4]:
```suggestion
```

**Review by SMillerDev — COMMENTED** [review]
Please add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap) here. The [CreateZap](https://github.com/nrlquaker/homebrew-createzap) tool can help you with this.

**Comment by execat** [comment]
Thanks for the changes!

One question: `brew createzap gdevelop` listed `"~/Documents/GDevelop projects"` as a thing to go in zap, but it might contain project data created by the user. So I left it out as preferred by https://docs.brew.sh/Cask-Cookbook#stanza-zap. Sounds good?

**Comment by execat** [comment]
Hi, can we approve/merge this since all comments are resolved?

**Comment by execat** [comment]
Thanks!

**Pull request merged**